SalesforceSDK/Universal/App/SFApplicationHelper.cs

```diff
@@ -116,6 +117,7 @@ private Frame CreateRootFrame()
                 {
                     throw new Exception("Failed to create initial page");
                 }
+                rootFrame.Navigate(typeof(AccountPage), e.Arguments);
```

Comment:
what did this navigate to before you added this line?  Or would it not navigate at all and let the consumer do a navigation?